packages/components/forms/src/form/Form.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: 'Form'
+type: 'component'
+status: 'stable'
+slug: /components/form/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/forms/src/form'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/form-elements-form--basic'
+---
+
+`Form` is a wrapper component to apply a `<form />` wrapper and handle form submissions.
+
+## Table of contents
+
+- [How to use Form](#how-to-use-Form)
+
+## How to use Form
+
+- The `Form` component should wrap form elements into `<form />`
+- Provide an `onSubmit` handler for form validation and submission
+- state of the validation of the form should be maniputated from outside
+
+### Basic example
+
+```jsx
+<Form onSubmit={() => console.log('handler for submit')}>
+  <FormControl>
+    <FormControl.Label isRequired>Name</FormControl.Label>
+    <TextInput aria-label="input" />
```

Comment:
`aria-label` is not needed here, because we have label in place

Reply:
yup, true. removed